Build.fsx

```diff
@@ -19,7 +19,9 @@ let build target configuration =
     let properties = [ "Configuration", configuration ] @ keyFile
 
     solutionsToBuild
-    |> MSBuild "" target properties
+    |> Seq.iter (fun s -> build (fun p -> { p with Verbosity = Some(Minimal)
```

Comment:
Why set the verbosity to minimal? Don't we want to have as much information as possible when troubleshooting?